api/src/main/java/keywhiz/api/model/SanitizedSecret.java

```diff
@@ -76,7 +76,7 @@ public static SanitizedSecret fromSecretSeriesAndContent(SecretSeriesAndContent
         series.type().orElse(null),
         series.generationOptions(),
         content.expiry(),
-        series.currentVersion().orElse(null));
+        content.id());
```

Comment:
This looks good.  This change ensures that the "version" of a SanitizedSecret is the ID associated with its contents.  In most cases, `content.id()` will match `series.currentVersion()`; however, when handling secret versions the content ID for past versions deliberately does not match the secret's current version ID.